pkg/diagnostics/systemd/unit_status.go

```diff
@@ -98,7 +98,7 @@ To ensure it is not failing to run, check the status and logs with:
   # journalctl -ru %[1]s`
 
 	sdUnitReqLoaded = `
-systemd unit %[1]s depends on unit %[2]s, which is not loaded.
+systemd unit %[1]s depends on unit %[2]s, which is not loaded.  %[3]s
```

Comment:
@deads2k this... doesn't look right. i'll make a fixup PR. don't need to keep this separate though.

https://github.com/deads2k/origin/pull/33